src/utils/beneficial.owner.statements.ts

```diff
@@ -94,3 +86,17 @@ export const postBeneficialOwnerStatements = async (req: Request, res: Response,
     next(error);
   }
 };
+const getRedirectUrl = (req: Request, registrationFlag: boolean, noChangeRedirectUrl?: string) => {
+  let REDIRECT_URL: string;
```

Comment:
Minor... Not sure why this variable was ever defined in capitals, but since code is changing, would be nice to redefine this as just `redirectUrl`. The definition higher up too, as even though that's a const it seems that we use camelCase elsewhere in the code-base.

Reply:
The change to camelCase has been applied.